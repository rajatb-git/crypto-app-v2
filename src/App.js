import React from 'react';

import Header from './components/Header/Header.jsx';

function App() {
  return (
    <div id="root" className="App">
        <Header />
    </div>
  );
}

export default App;
