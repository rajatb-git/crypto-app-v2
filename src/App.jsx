import React from 'react';
import { BrowserRouter as Router } from 'react-router-dom';

import Routing from './Routing/Router';
import { Header } from './components';
import Firebase, { FirebaseContext } from './firebase';

const App = () => (
  <FirebaseContext.Provider value={new Firebase()}>
    <Router>
      <Header />
      <Routing />
    </Router>
  </FirebaseContext.Provider>
);

export default App;
