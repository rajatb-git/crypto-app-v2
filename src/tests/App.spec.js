import React from 'react';
import App from '../App.js';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  expect(renderer.create(<App />)).toMatchSnapshot();
});
