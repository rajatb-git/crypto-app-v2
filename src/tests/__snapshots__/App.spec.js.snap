// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="App"
  id="root"
>
  <nav
    className="navbar navbar-expand-lg navbar-dark bg-dark"
  >
    <a
      className="navbar-brand"
      href="#home"
    >
      Crypto App
    </a>
    <button
      aria-controls="basic-navbar-nav"
      aria-label="Toggle navigation"
      className="navbar-toggler collapsed"
      onClick={[Function]}
      type="button"
    >
      <span
        className="navbar-toggler-icon"
      />
    </button>
    <div
      aria-expanded={null}
      className="navbar-collapse collapse"
      id="basic-navbar-nav"
    >
      <div
        className="mr-auto navbar-nav"
        onKeyDown={[Function]}
      >
        <a
          class="nav-link"
          href="/"
          onClick={[Function]}
        >
          Home
        </a>
        <a
          class="nav-link"
          href="/admin"
          onClick={[Function]}
        >
          Admin
        </a>
      </div>
    </div>
  </nav>
  <div>
    <h2>
      Landing
    </h2>
  </div>
</div>
`;
